dbt/adapters/athena/impl.py

```diff
@@ -631,6 +631,18 @@ def list_schemas(self, database: str) -> List[str]:
             result.extend([schema["Name"] for schema in page["DatabaseList"]])
         return result
 
+    @staticmethod
+    def _is_current_column(col: dict) -> bool:
+        """
+        Check if a column is current, it always fallback to True if the parameter is not property is not available
+        """
+        if col.get("Parameters", {}).get("iceberg.field.current") == "true":
+            return True
+        elif col.get("Parameters", {}).get("iceberg.field.current") == "false":
+            return False
+        else:
+            return True
```

Comment:
```suggestion
        if col.get("Parameters", {}).get("iceberg.field.current") == "false":
            return False
        return True
```

Reply:
does it make sense to simplify the logic a bit ?

Reply:
We can, I just wanted to be "explicit", but your suggestion is more elegant.